react/features/video-quality/reducer.js

```diff
@@ -69,9 +69,29 @@ function _setConfig(state, { config }) {
     const configuredMap = config?.videoQuality?.minHeightForQualityLvl;
     const convertedMap = validateMinHeightForQualityLvl(configuredMap);
 
+    const heightByParticipant = config?.videoQuality?.restrictVideoHeightByParticipants;
+    const heightByParticipantMap = validateRestrictVideoHeightByParticipants(heightByParticipant);
+
     if (configuredMap && !convertedMap) {
         logger.error('Invalid config value videoQuality.minHeightForQualityLvl');
     }
 
-    return convertedMap ? set(state, 'minHeightForQualityLvl', convertedMap) : state;
+    logger.error(heightByParticipant);
+    logger.error(heightByParticipantMap);
```

Comment:
looks like these log outputs got here by accident

Reply:
yes, those need to be removed